packages/react-components/src/molecules/AnalyticsControls.tsx

```diff
@@ -150,6 +156,21 @@ const updateSearchParams = (): URLSearchParams => {
   return searchParams;
 };
 
+const getLastUpdate = () => {
+  const lastUpdate = new Date();
+  lastUpdate.setUTCHours(6, 0, 0);
+
+  const now = new Date();
+
+  if (lastUpdate.getUTCHours() >= now.getUTCHours())
```

Comment:
I think you can remove the `.getUTCHours()` from both `lastUpdate` and `now` as we just need to check that we're not setting `lastUpdate` to a future date, not just the time.

Reply:
If its currently 6:15am UTC we would still want to set the previous days sync time even though lastUpdated is not a future date since the sync wouldn't be done running and the data seen in the hub would be from the previous day.

Reply:
Hmmm...then I think you need to represent some offset to consider the time it takes to update rather than comparing the hours directly, no?

Reply:
I'm not sure I understand why not to compare the hours directly. The sync takes about an hour (I added this comment earlier but it's outdated since I pushed another commit https://github.com/yldio/asap-hub/pull/4365#discussion_r1743402161). Since last update is 6am, if the current hour now is 6 or less then set date to previous day.